kafka-exporter/src/test/java/io/opentelemetry/contrib/kafka/KafkaSpanExporterIntegrationTest.java

```diff
@@ -38,17 +38,18 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInstance;
 import org.rnorth.ducttape.unreliables.Unreliables;
-import org.testcontainers.containers.KafkaContainer;
 import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.kafka.ConfluentKafkaContainer;
 import org.testcontainers.utility.DockerImageName;
 
 @Testcontainers(disabledWithoutDocker = true)
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class KafkaSpanExporterIntegrationTest {
   private static final DockerImageName KAFKA_TEST_IMAGE =
-      DockerImageName.parse("confluentinc/cp-kafka:6.2.1");
+      DockerImageName.parse("confluentinc/cp-kafka:7.4.0")
+          .asCompatibleSubstituteFor("apache/kafka");
   private static final String TOPIC = "span_topic";
-  private KafkaContainer kafka;
+  private ConfluentKafkaContainer kafka;
```

Comment:
I'd use `org.testcontainers.kafka.KafkaContainer` and set the image to `apache/kafka:3.8.0`. That is how we handled this in the instrumentation repo.

Reply:
Thanks will validate this and update accordingly

Reply:
@vincentfree you may the permissions needed to push directly to this PR, but if you do find changes to make, just send a new PR, and renovate will automatically close this PR once we merge yours

Reply:
I will implement the suggestions posted by @laurit, using the vendor neutral kafka variant should be preferred over the use of the confluent variant.